Multisig/UI/Safe Management/Create Safe/CreateSafeFlow.swift

```diff
@@ -68,16 +77,38 @@ class CreateSafeFlow: UIFlow {
     }
 
     func appleLogin() {
-        // TODO: Create login via apple
-        // This line is not needed after this method implementation
-        creatingSafe()
+        handleAuthorizationAppleIDButtonPress()
+
         // TODO: submit create safe tx
     }
 
+    func handleAuthorizationAppleIDButtonPress() {
+        let appleIDProvider = ASAuthorizationAppleIDProvider()
+        let request = appleIDProvider.createRequest()
+        request.requestedScopes = [.fullName, .email]
+
+        appleWeb3AuthLogin = AppleWeb3AuthLogin (
+            authorizationComplete: {
+                let view = SafeCreatingViewController()
+                view.onSuccess = { self.safeCreationSuccess() }
+                self.show(view)
+            }, keyGenerationComplete: { key in
+                print("key: \(key)")
```

Comment:
This line doesn't seem like continuing with the flow on success 🤔

"Continue after successful key generation" for apple login - will it be implemented in this pr or it was not an intention to do it here?

Reply:
This should mark the place where safe creation can start. I added a TODO